.travis.yml

```diff
@@ -2,5 +2,9 @@ language: ruby
 sudo: false
 cache: bundler
 rvm:
-  - 2.0.0
-script: 'bundle exec rake ci'
\ No newline at end of file
+- 2.0.0
+script: bundle exec rake ci
+env:
+  global:
+  - secure: paT789bDLaYTixyufAu1RDHOEjXPETsKC+crAMAHHy3YWyhhys7uJ3d4rzOwkDED3WD0hLB17JqyqYVysDkn+hFWPdGbHwgtTVGUVJaDMa9kfzsR/dQuCF0z6vSsGDqIGMXlsTPWYZmoU8QPWwtewA21r7QPJnXcsO/kr9kgedFKAogwmKGq/7MWQ8+2ib0x6f36M8mcW6eJ4baBOeu9tXdgUEErnDx1Tx+UK37c0wobu0H5i4Kq0P7auEE4nuGyHhV8cnFhSKz89KNDniUNfEGC9Vtucbnb8czmEFcQx77yrwYIzylEzqoVCK/8KRHKAN9fJ5wu+Eg/MPuiYEEs63lrQJfiP9zZKp7mX5MrqTJMmOysgYQDKP7qGzDD1tSmZxqd4aRQJ8ba36nW96udLnUvEDkpDB6r9OwE82i2aaLfnOIYx8SsT3pT4v1RPi+cpqmpnpGRxyMVqZfE23h7oIVhgrwjSYmOP0sRbiUbe8xjRLxGuBM3Z8E85xYNvz0Xd/sAnPnricOsagXk/5RJ11fnOEjO26LahroANAzdJNo6Ovb1hL/oC9ghvNWJo/52Y2Bgtz6M3Lb2NY22W9id2TpN9RleXTEAdApUDyxtSIDLsQFtmOXlEh1X1kXLr19ADwdtsc0zRJHgu3qExWGfAHw4RXLz/DtkfnHyyqg/qDc=
+  - secure: rpBvnyOEcpkuOPS/a6YGMDRla8zMi5WKcQYFoTSaNeChdKAUIiKxakcZsaAua2uqog6Wh+eS69/4vUJ1MDxUcKpMP5QosAC0VyKrTzPUrm464xeo2ZVYsyWVDV42gDhM2ahuWeQW9NS5YiUDbl9/UaeFPFYwUNYcNZqr/uVU2Sy7NSkvN/68CuMqztAmagqZkcCy7kI6tV2ogFd5g7DFb/NfMpx9YHmYNlhbzDOTvsHjDPW49FUoNpxPnT8K17+kYs2tZRdEyyjSjxS2h45qLTy2kGV3mfTzIGOw3+3Q75kEYSn/EgSaMuEX59uBHCLAc6xO3t7TznMEP/vBhTRcaAVK500MmKGJ9cUnqnnFg05VufrTnbm38ryqurjVmb6ks5GQdLWkb3Kl+zLsdJ7H8f9ZaBMhMtjZADMeHfpzfiOgM5TUVBTDPPtoVH6qAyBj2BieUP5ms2HX6CVNKZ41et9ZvmTkBVNeEB1E86fZzfJ2n05dq+7sUH1wHeN3m9jrOKk7wnPDfdvCgPEjBwGb41Lk1soev4nXgo9NJY/KkMycWoBJFasdENULDwZ65MEVM6+wAahv9luh8o4/GiIFEe3MzPckMx+uPFxHenC8WFAmBlrG42Y9cFkImCGccrnggg3fTNC/ZUoYR88B2zLTFSymMfvkMo1m5dLoSEsalWM=
```

Comment:
@afred: Do there really need to be two `secure`s? Could you provide a comment here about how these are (re)generated?